package.json

```diff
@@ -41,6 +40,9 @@
     "tsify": "^5.0.2",
     "typescript": "^4.3.5"
   },
+  "peerDependencies": {
+    "cypress": "*"
```

Comment:
Can you restrict the Cypress version to be `>= 7.0` as I believe there were some changes made between v6 and v7.

Reply:
Done.